pages/folder/[id].tsx

```diff
@@ -98,22 +99,10 @@ function Folder() {
         />
       ) : null}
 
-      <F.Main ref={targetRef}>
-        {scrollAddUrl ? (
-          <F.addLink>
-            <img src="/img/link.png" alt="linkimg" />
-            <F.addInput placeholder="링크를 추가해 보세요" />
-            <F.addButton>추가하기</F.addButton>
-          </F.addLink>
-        ) : null}
-      </F.Main>
+      <F.Main ref={targetRef}>{scrollAddUrl ? <AddLink /> : null}</F.Main>
```

Comment:
```suggestion
      <F.Main ref={targetRef}>{scrollAddUrl && <AddLink />}</F.Main>
```
&& 연산자를 이용하면 불필요한 null을 입력하지 않아도 됩니다